File: python/led/led.py
import mraa
import time

x = mraa.Gpio(8)
x.dir(mraa.DIR_OUT)

while True:
    x.write(1) 
    print 'LIGOU'
    time.sleep(0.2)
    x.write(0)
    print 'DESLIGOU'
    time.sleep(0.2)

File: c++/led/led.c
#include "mraa.h"
#define DEFAULT_IOPIN 9

static int iopin;

int main(int argc, char** argv)
{
  iopin = DEFAULT_IOPIN;
  mraa_gpio_context gpio;
  mraa_init();
  gpio = mraa_gpio_init(iopin);
  mraa_gpio_dir(gpio, MRAA_GPIO_OUT);
  while(1){
    mraa_gpio_write(gpio, 1);
    sleep(1);
    mraa_gpio_write(gpio, 0);
    sleep(1);
  }
   return 0;
}
